info.yml

```diff
@@ -100,7 +100,7 @@ configuration:
                       specifies the shotgun entity type to display. *filters* is a list of standard API Shotgun 
                       filters. *hierarchy* is a list of shotgun fields relative to the entity type, defining the 
                       grouping of the tree."
-        allows_empty: False
+        allows_empty: True
```

Comment:
There's no reason why a TD shouldn't be able to configure the app to only have the My Tasks tab.